internal/envoy/shutdown/admin_test.go

```diff
@@ -74,5 +62,5 @@ func runWithShutdownHook(t *testing.T, runDir string, hook func(r *envoy.Runtime
 		if fakeInterrupt != nil {
 			fakeInterrupt()
 		}
-	}, r, stderr, args...)
+	}, r, stderr, "-c", "envoy.yaml")
```

Comment:
fake envoy should crash if config isn't passed. this was crashing but we never checked the error